src/main/java/com/twilio/sdk/resource/instance/taskrouter/TaskQueueStatistics.java

```diff
@@ -98,6 +101,36 @@ public TaskQueueStatistics(final TwilioTaskRouterClient client, final String wor
 	}
 
 	/**
+	 * Get the number of tasks with the Priority
+	 * @return number of tasks with Priority priority
+	 */
+	public Integer getTasksWithPriority(int priority) {
+		Map<String, Object> tasksByPriority = (Map<String, Object>) getRealtime().get(TASKS_BY_PRIORITY_PROPERTY);
+
+		String tasksByPriorityKey = "Priority-" + String.valueOf(priority);
+
+		Integer tasksCount = (Integer) tasksByPriority.get(tasksByPriorityKey);
+
+		return tasksCount != null ? tasksCount : 0;
```

Comment:
if we are going to guarantee that this will never be null, i would recommend returning a primitive int.